servatrice/src/serversocketinterface.cpp

```diff
@@ -849,7 +917,7 @@ Response::ResponseCode ServerSocketInterface::cmdBanFromServer(const Command_Ban
     QList<QString> moderatorList = server->getOnlineModeratorList();
     QListIterator<QString> modIterator(moderatorList);
     foreach(QString moderator, moderatorList) {
-        QString notificationMessage = "A ban has been put in with the following details:";
+        QString notificationMessage = moderator + "has placed a ban with the following information:";
```

Comment:
This doesn't look right... It seems like each mod will see that they placed the ban.